ts/components/ui/Markdown/handlers/internalLink.ts

```diff
@@ -13,7 +13,7 @@ export const IO_INTERNAL_LINK_PREFIX = "ioit://";
 const ALLOWED_ROUTE_NAMES: ReadonlyArray<string> = [
   ROUTES.MESSAGES_HOME,
   ROUTES.PREFERENCES_HOME,
-  ROUTES.PREFERENCES_SERVICES,
+  ROUTES.SERVICES_HOME,
```

Comment:
With this change you are breaking old messages links to services screen. We need to still handle old routename but forward to the new route.

Reply:
fixed in 13a1a6c2